src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/validator/TransactionInputValidator.java

```diff
@@ -33,4 +32,20 @@ public void validatePendingStatus(final List<TransactionDto> transactionDtos) {
 			throw new ApiRequestValidationException(errors.getAllErrors());
 		}
 	}
+
+	public void validateAllProvidedTransactionsExists(final List<TransactionDto> transactionDtos, final Set<Integer> transactionIds) {
+		errors = new MapBindingResult(new HashMap<String, String>(), TransactionDto.class.getName());
+		if (transactionIds != null && !transactionIds.isEmpty() && transactionIds.size() != transactionDtos.size()) {
+			errors.reject("transactions.does.not.exist", "");
```

Comment:
@cuenyad Should include information about the non valid ids, please check TransactionUpdateRequestDtoValidator, line 71. 
To keep standards